rdt-open-reader/src/main/res/values/strings.xml

```diff
@@ -53,5 +53,8 @@ Please click on the <i>Locate Me</i> button at the bottom of the screen and we w
     <string name="rdt_not_supported">Sorry, the selected RDT is not currently supported.</string>
     <string name="male">Male</string>
     <string name="female">Female</string>
+    <string name="lbl_back">Back</string>
+    <string name="lbl_gender_id">Female . ID: 99358357</string>
+    <string name="lbl_visit">Visit</string>
```

Comment:
Aren't these duplicates https://github.com/onaio/rdt-standard/pull/648/files#diff-1a659ac4e58d657a59f0333681eb88c15c48f0bb32a9a38b51b9c72de466009dR32?

Reply:
this command failed if remove these strings
`./gradlew :rdt-open-reader:clean :rdt-open-reader:testMalariaKenyaDebugUnitTestCoverage`